app/arm_driver/src/main.cpp

```diff
@@ -53,14 +53,16 @@ DigitalOut *dir_out[NUM_MOTORS];
 float freq = 50.0; // PWM Frequency [Hz]
 float period = 1 / freq;
 DigitalOut led1(PC_1);
+DigitalOut led2(PC_2);
 
 //QEI library used to read from incremental encoders.
 //To-Do : Determine the pins used for the encoders
-QEI turntable_inc(A0, A1, A2, 48);
-QEI wrist_pitch_inc(A0, A1, A2, 48);
-PwmIn wrist_roll_abs(A4);
-PwmIn shoulder_abs(A4);
-PwmIn elbow_abs(A5);
+//Pins and connectors correspond to Arm board rev_2
```

Comment:
for QEI encoders (incremental), are they actually 48 pulses per rev? also I think we need to set the encoding parameter. by default its only X2 encoding, but I think we should be using X4 encoding